v2/go.sum

```diff
@@ -1,6 +1,6 @@
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
-github.com/IBM/go-sdk-core/v4 v4.8.1 h1:gFcQcQGOoHg+8pNILVTgGMh77SG756XkOatv/gs2VTg=
-github.com/IBM/go-sdk-core/v4 v4.8.1/go.mod h1:GECJ/p0r9Hs0XcOnCiGjZrt6M/rQc+gW8YkCsGJ+j5U=
+github.com/IBM/go-sdk-core v0.0.0-20200217212347-fe152a0e9e89 h1:xvNuMMhsiGhHUA7DWob/sM8OFddqDdzOJkpX626cA1I=
```

Comment:
this doesn't look right to me.  we shouldn't have any code in your project that imports the go core without the "v4" part being included in the import path.